wallet/src/de/schildbach/wallet/ui/dashpay/DashPayViewModel.kt

```diff
@@ -68,7 +67,7 @@ class DashPayViewModel(application: Application) : AndroidViewModel(application)
         }
     }
 
-    fun searchUsernames(text: String, userId: String) {
+    fun searchUsernames(text: String, userId: String = "") {
```

Comment:
**userId** is the current **userId** of the user of the app.  This value is required to get all contacts, both the contacts that this user has created and that other users have created with this user.